components/TabPanel.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from '../pages/Transactions/Transactions.module.css';
```

Comment:
Just to keep things organized, move this TabPanel component into its own folder (like the SideNavbar component)